### DISCUSSION TITLE
Not downloaded beatmap(s) preview in playlists

### DISCUSSION THREAD

**Discussion by FriendsNone** [ppy/osu#21315] [Ideas] [answered]
Currently (version 2022.1117.0), selecting a beatmap that you haven't downloaded yet in playlists gives you silence. It's somewhat of a hassle right clicking, going into details just to play the preview. Or downloading it straight up.

Whether it's a bug or not yet implemented. I propose beatmap previews (one from the listings) to be played while selecting a not downloaded beatmap. An extra play/pause button could also be added next to leaderboards and download.

**Comment by peppy** [comment]
This does sound like it could be a good addition.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/28333

**Comment by Ladwick** [comment]
Last week I was thinking in exactly this, it would be really cool specially for people that goes to the playlists to download beatmaps! (like me)
